doc/api/cli.md

```diff
@@ -100,6 +100,34 @@ If this flag is passed, the behavior can still be set to not abort through
 [`process.setUncaughtExceptionCaptureCallback()`][] (and through usage of the
 `node:domain` module that uses it).
 
+### `--build-snapshot`
```

Comment:
I think prefixing the option would be a bit weird - it has been used internally for quite sometime, and also many flags have been experimental without the prefix, making the prefix superfluous and it unnecessarily hinders adoption. Marking it as experimental in the doc should be enough, IMO.

Reply:
Both sound good to me. I was thinking that people can easily crash the process when building the snapshot (like signal listeners) or resurrecting from the snapshot because of unsupported features. An explicitly warn about the immaturity of the feature can be helpful.

Reply:
That's fair, I will add a note in the doc warning about this, though I would not worry too much about the crashing because there are many flags that can lead to a crash (e.g. setting the old space size to a very small value)